pig_latin.py

```diff
@@ -0,0 +1,13 @@
+import re
+
+def pig_it(text):
+    words = text.split(' ')
+    result = []
+
+    for word in words:
+        if re.match('^[A-Za-z]+$', word):
+            result.append(word[1:] + word[0] + 'ay')
+        else:
+            result.append(word)
```

Comment:
What would this look like as a list comprehension?

Reply:
```
words = s.split()
result = [word[1:] + word[0] + 'ay' if re.match('^[A-Za-z]+$', word) else word for word in words]
return ' '.join(result)
```